src/index2.md

```diff
@@ -99,8 +99,12 @@ layout: home2
         <p class="Home2-testimonial-profileInfo-role">Software Eng. Lead</p>
       </div>
       <div class="Home2-testimonial-profileSocial">
-        <a href="https://twitter.com/theotherelliott" rel="noopener noreferrer" target="_blank"  class="Home2-testimonial-profileSocial-link" >{% svg assets/svg/social-twitter.svg %}</a>
-        <a href="http://engblog.yext.com/author/telliott" rel="noopener noreferrer" target="_blank" class="Home2-testimonial-profileSocial-link">{% svg assets/svg/social-web.svg %}</a>
+        <a href="https://twitter.com/theotherelliott" rel="noopener noreferrer" target="_blank"  class="Home2-testimonial-profileSocial-link" >
+          {% svg assets/svg/social-twitter.svg class="Home2-testimonial-profileSocial-svg Home2-testimonial-profileSocial-svg--twitter" %}
+        </a>
+        <a href="http://engblog.yext.com/author/telliott" rel="noopener noreferrer" target="_blank" class="Home2-testimonial-profileSocial-link">
+          {% svg assets/svg/social-web.svg class="Home2-testimonial-profileSocial-svg Home2-testimonial-profileSocial-svg--web" %}
```

Comment:
While I was in here, made the web icon more consistent (set color via CSS instead of in the SVG)